simapp/app_legacy.go

```diff
@@ -648,6 +653,10 @@ func (app *SimApp) RegisterTendermintService(clientCtx client.Context) {
 	)
 }
 
+func (app *SimApp) RegisterNodeService(clientCtx client.Context) {
+	nodeservice.RegisterNodeService(clientCtx, app.GRPCQueryRouter())
+}
+
```

Comment:
Not really since different services might require different arugments/deps (e.g. tmservice).

What we could perhaps explore is putting this all into a single function that apps can call?